bubblesub/data/options.yaml

```diff
@@ -32,6 +35,12 @@ gui:
     splitters: {}
     fonts: {}
 
+    visibility: {
+        "console": false,
```

Comment:
Let's make sure the console is visible by default, as it's pretty important feature that can be always hidden at a later time. Having this default to false destroys its discoverability.

Reply:
I thought the console was like the log dialog, usually is disabled by default in other GUI programs, that's why I set it to false